openshift/release/update-faas.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+
+readonly ROOT_DIR=$(dirname $0)/../..
+
+FAAS_VERSION=${1:-"main"}
```

Comment:
IMO we should not use any external arguments, but hardcode the versions in the scripts or at least used kind of 'named' arguments like in 

```
FAAS_VERSION=${$FAAS_VERSION:-"main"}
```

so that it could be called with 

```
FAAS_VERSION=test ./update-faas.sh
```

Reply:
Ok, I wasn't sure how we'd like to invoke the script, but your suggestions sounds good.